source/store/AuthContext.js

```diff
@@ -143,16 +158,19 @@ function AuthProvider({ children, initialState }) {
   /**
    * This function checks if the current accessToken is valid.
    */
-  async function isAccessTokenValid() {
+  const isAccessTokenValid = useCallback(async () => {
     const decodedToken = await authService.getAccessTokenFromStorage();
 
     // Configure app in dev mode if personalnumber is defined as test account
-    if (decodedToken?.personalNumber && env.TEST_PERSONAL_NUMBER === decodedToken.personalNumber) {
-      handleSetMode('development');
+    if (
+      decodedToken?.personalNumber &&
+      env.TEST_PERSONAL_NUMBER === decodedToken.personalNumber
+    ) {
+      handleSetMode("development");
     }
 
     // TODO: Remove this condition when exp value is set on the jwt token in the api.
```

Comment:
TODOs should not ever be commited
I know, this is not on your  hands :)

Reply:
Will not change this